### ISSUE TITLE
Scripts not opening in current project directory

### ISSUE THREAD

**Issue by grantmcdermott** [julia-vscode/julia-vscode#1437]
As I read https://github.com/julia-vscode/julia-vscode/pull/690 any Project.toml file should act as a relative path anchor.

However, whenever I open up a Julia script (which is part of a project) in VS Code and try to run it interactively, I find that I'm automatically taken to my home directory. So I first end up having to `cd` over to the project directory manually and only then can I activate the project environment, start running code, etc.

Am I missing something obvious? Should I be doing something other than opening up VS Code by clicking on the .jl script?

FWIW, my simplified directory structure might look something like this:

```
home/
 |_ projectdir/
     |_ Manifest.toml
     |_ Project.toml
     |_ myscript.jl
```

**Comment by davidanthoff** [comment]
The working folder for the Julia REPL will be the root of your workspace that is open in VS Code. You probably want to open a folder in VS Code, not an individual file.

We should probably look through the behavior that we get if you open just a file, not a folder, I'm actually not sure what determines what is happening in that situation.

**Comment by grantmcdermott** [comment]
Thanks David, that's certainly helpful.

On second thought, I'm not sure how a Project.toml would reference against individual files. But my ideal would still be that opening up a .jl script in VS Code would set the relative path (e.g. like R and other SW).

**Comment by davidanthoff** [comment]
Yeah, I think if you open a single file (not a workspace/folder), we could set the working folder to the parent folder of that file, that seems to make sense to me.
